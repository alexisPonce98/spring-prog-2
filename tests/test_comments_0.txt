z, u;
v0{
  public: v;
  private: x, y;
    v1{
      private: x, v;
      x = y;
      v = w;
      v2{
      	public: w;
      	private: z,y;
      	w=v;
      	x=z;
      	y=u;
      }
      x=y;
      w=v;
    }
    x = y;
    u = z;
    v = w;
} 
