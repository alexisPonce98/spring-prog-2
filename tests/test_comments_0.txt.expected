v1.x = ?.y
v1.v = ?.w
v2.w = v0.v
?.x = v2.z
v2.y = ::u
v1.x = ?.y
?.w = v1.v
v0.x = v0.y
::u = ::z
v0.v = ?.w

